.github/workflows/build-web-extension.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Web Extension
+
+on:
+    workflow_dispatch:
+
+    push:
+        branches:
+            - "v4"
+            - "master"
+        paths:
+            - "packages/app/**"
+            - "packages/core/**"
+            - "packages/extension/**"
+            - "packages/locale/**"
+
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: actions/setup-node@v1
+              with:
+                  node-version: 16.13.1
+            - name: Install dependencies
+              run: |
+                  npm ci
+                  npm i -g web-ext@6.6.0
+            - name: Build
+              run: npm run web-extension:build
+            # - name: Sign for Firefox
+            #   run: cd packages/extension/dist && web-ext sign --api-key=${{ secrets.WEB_EXT_API_KEY }} --api-secret=${{ secrets.WEB_EXT_API_SECRET }}
```

Comment:
@MaKleSoft can you please set these two secrets in GitHub so I can uncomment these two steps?